components/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import facebookImg from "@/public/img/png/Vector-1.png";
+import twitterImg from "@/public/img/png/Vector-2.png";
+import youtubeImg from "@/public/img/png/Vector-3.png";
+import melonImg from "@/public/img/png/Vector-4.png";
+import Image from "next/image";
+import styles from "./footer.module.css";
+
+const SNS = [
+  {
+    title: "페이스북",
+    image: facebookImg,
+    link: "https://www.facebook.com/",
+  },
+  {
+    title: "트위터",
+    image: twitterImg,
+    link: "https://twitter.com/",
+  },
+  {
+    title: "유튜브",
+    image: youtubeImg,
+    link: "https://www.youtube.com/",
+  },
+  {
+    title: "멜론",
+    image: melonImg,
+    link: "https://www.melon.com/",
+  },
+];
+
+const Footer = () => {
+  return (
```

Comment:
styles에서 여러 className을 가져오고 있으므로 구조분해할당을 사용해 주시면 좋을 것 같아요

```suggestion
const { footer, footerConnect } = styles;
```